yarn.lock

```diff
@@ -1653,15 +1653,15 @@ __metadata:
   linkType: hard
 
 "@npmcli/agent@npm:^2.0.0":
-  version: 2.2.0
-  resolution: "@npmcli/agent@npm:2.2.0"
+  version: 2.2.1
```

Comment:
Not sure why pkg versions were updated, as all packages other than `typescript@<5.2.0` use fixed version numbers - `yarn install` shouldn't have done anything. Maybe the non-fixed TS pkg version is affecting this? :thinking: